libs/core/tests/unit_tests/runnables/test_runnable.py

```diff
@@ -1111,6 +1117,7 @@ def test_configurable_fields() -> None:
             "lang": {"title": "Lang", "type": "string"},
             "name": {"title": "Name", "type": "string"},
         },
+        "required": ["lang", "name"],
```

Comment:
Great! This looks like it's improving the schema that we get as its more precise about what's required and what's optional!